src/Microsoft.TestPlatform.Common/Utilities/FakesUtilities.cs

```diff
@@ -24,7 +24,9 @@ public static class FakesUtilities
 
         private const string CrossPlatformConfiguratorMethodName = "GetCrossPlatformDataCollectorSettings";
 
-        private const string FakesConfiguratorAssembly = "Microsoft.VisualStudio.TestPlatform.Fakes, Version=16.0.0.0, Culture=neutral";
+        // Must be kept in sync with version being generated by VSUnitTesting
+        private const string version = ExternalAssemblyVersions.MicrosoftFakesAssemblyVersion;
+        private const string FakesConfiguratorAssembly = "Microsoft.VisualStudio.TestPlatform.Fakes, Version=" + version + ", Culture=neutral";
```

Comment:
I assume it will not compile as `FakesConfiguratorAssembly` is const and `version`'s  value is not known as compile time.

Reply:
Actually, it built fine :)

Reply:
It's known during compile time as the assembly version is a hard coded const string, and not really acquired through reflection. We need to do this as the `TestPlatform.Fakes` assembly is dropped by the VSUnitTesting build into VS and not from this repo